scripts/setup.sh

```diff
@@ -14,34 +14,72 @@ stack_resolver() {
   return 0
 }
 
-setup() {
-  # $1 - optional - path to haskell-vim-now installation
-  #      If not set, script will use default location.
-  unset HVN_DEST
-  local HVN_DEST
-  if [ -z $1 ] ; then
-    # No argument provided, using default path
-    HVN_DEST="$(config_home)/haskell-vim-now"
+setup_haskell() {
+  local HVN_DEST=$1
+  local RETCODE
+
+  if ! check_exist stack >/dev/null ; then
+    err "Installer requires Stack."
+    msg "Installation instructions: https://github.com/commercialhaskell/stack#how-to-install"
```

Comment:
While you're at it, this url has moved to http://docs.haskellstack.org/en/stable/README/#how-to-install